src/js/utils/SchemaUtil.js

```diff
@@ -26,13 +26,12 @@ function setLabelFromSchemaProperty(fieldName, fieldProps, isRequired, renderLab
   }
 
   if (renderLabel && fieldProps.description) {
-    label = renderLabel(fieldProps.description, label);
+    label = renderLabel(fieldProps.description, label, fieldProps.type);
   }
 
   // Set the label property of checkboxes if a label is defined.
-  if (fieldProps.type === 'boolean' && fieldProps.label != null
-    && fieldProps.label !== '') {
```

Comment:
because even if there are no `label` property on the schema, we can still have a label from the `title` property, which we set above